src/Migrations/M040MigrateArticlesBlockToPostsListBlock.php

```diff
@@ -0,0 +1,356 @@
+        $layout['type'] = $layout_type;
+        $layout['columnCount'] = 4;
+
+        $attrs = [];
+        $attrs['queryId'] = 0;
+        $attrs['query'] = $query;
+        $attrs['namespace'] = Utils\Constants::BLOCK_POSTS_LIST;
+        $attrs['layout'] = $layout;
+        return $attrs;
+    }
+
+    /**
+     * Create and get a new buttons container block.
+     *
+     * @return array - The new block.
+     */
+    private static function get_buttons_block(): array
+    {
+        return Utils\Functions::create_block_buttons(
+            [
+                'lock' => [
+                    'move' => true,
+                ],
+                'layout' => [
+                    'type' => 'flex',
+                    'justifyContent' => 'space-between',
+                    'orientation' => 'horizontal',
+                    'flexWrap' => 'nowrap',
+                ],
+                'className' => 'carousel-controls',
+            ],
+            [
+                Utils\Functions::create_block_single_button(
+                    ['className' => 'carousel-control-prev'],
+                    __('Prev', 'planet4-blocks'),
+                ),
+                Utils\Functions::create_block_single_button(
+                    ['className' => 'carousel-control-next'],
+                    __('Next', 'planet4-blocks'),
+                ),
+            ]
+        );
+    }
+
+    /**
+     * Create and get a new post template.
+     *
+     * @return array - The new template.
+     */
+    private static function get_post_template(): array
+    {
+        return Utils\Functions::create_post_template(
+            [
+                Utils\Functions::create_block_columns(
+                    [],
+                    [
+                        Utils\Functions::create_new_block(
+                            Utils\Constants::BLOCK_FEAT_IMAGE,
+                            ['isLink' => true]
+                        ),
+                        Utils\Functions::create_group_block(
+                            [
+                                Utils\Functions::create_group_block(
+                                    [
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'post_tag', 'separator' => ' ']
+                                        ),
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'category', 'separator' => ' | ']
+                                        ),
+                                    ],
+                                    ['className' => 'posts-list-meta']
+                                ),
+                                Utils\Functions::create_new_block(
+                                    Utils\Constants::BLOCK_TITLE
```

Comment:
This Post title is a link

Reply:
Post title link updated. Please have a look.